### PR TITLE
CORE-5122 Update signer summary hash calculation

### PR DESCRIPTION
**Pull request by kyriathar**
Changes cert summary hash calculation to use X500 names instead of public keys.

### CODE REVIEW

File: libs/packaging/packaging/build.gradle [hunk 1]
```diff
@@ -65,7 +69,8 @@ def certificateProvider = provider {
     def cert = file('config/dev/corda_dev_cpk.cer').withInputStream {
         cf.generateCertificate(it)
     }
-    cert.publicKey.encoded.sha256()
+    Base64.getEncoder().encodeToString(cert.encoded)
+//    cert.publicKey.encoded.sha256()
```
Comment (josephzunigadaly) [inline libs/packaging/packaging/build.gradle]: Let's delete this commented out code

Comment (kyriathar) [reply libs/packaging/packaging/build.gradle]: Removed

File: libs/packaging/packaging/build.gradle [hunk 2]
```diff
@@ -65,7 +69,7 @@ def certificateProvider = provider {
     def cert = file('config/dev/corda_dev_cpk.cer').withInputStream {
         cf.generateCertificate(it)
     }
-    cert.publicKey.encoded.sha256()
+    Base64.getEncoder().encodeToString(cert.encoded)
```
Comment (josephzunigadaly) [inline libs/packaging/packaging/build.gradle]: Can we move this to become a resource in the test folder? 

The only usage I can see is passing it into the test as a system property.

Comment (kyriathar) [reply libs/packaging/packaging/build.gradle]: Done

**Review by josephzunigadaly — APPROVED** [review]

**Pull request merged**